src/main/java/Action.java

```diff
@@ -0,0 +1,150 @@
+import java.lang.reflect.Array;
+import java.util.*;
+
+public class Action {
+    private String commandName;
+    private Item item;
+    private Room room;
+
+    public Action(String commandName, Item item, Room room) {
+        this.commandName = commandName;
+        this.item = item;
+        this.room = room;
+    }
+
+    public static String printInventory(ArrayList<Item> playerItems) {
+        String playerInventory = "";
+        String playerInventoryFinal;
+
+        for (Item item : playerItems){
+            playerInventory = item.getNameFromItem() + ", " + playerInventory;
+        }
+        int unnecessaryComma = playerInventory.lastIndexOf(",");
+        playerInventoryFinal = "Your current inventory: " + playerInventory.substring(0, unnecessaryComma);
+
+        return playerInventoryFinal;
+    }
+
```

Comment:
The use of two different variables here is unnecessary as this could have been done with only playerInventory. Other than that this seems fine.